test/sim/decisions.js

```diff
@@ -484,6 +484,49 @@ describe('Choices', function () {
 		});
 	});
 
+	describe("Rotation requests", function () {
+		it("should allow rotations in rotation battles", function () {
+			battle = common.gen(5).createBattle({gameType: 'rotation'}, TRIPLES_TEAMS.full);
+
+			battle.makeChoices('rotate right move tackle', 'rotate left move tackle');
+			assert.species(battle.p1.active[0], "Ivysaur");
+			assert.species(battle.p2.active[0], "Blastoise");
```

Comment:
Since you are using ``TRIPLES_TEAMS``, could you unhardcode the species and refer to them by their array indices?

```js
TRIPLES_TEAMS.full[0][2]
TRIPLES_TEAMS.full[1][1]
```